<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminInterace</class>
 <widget class="QDialog" name="AdminInterace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(:/img/curtains.png);
}
QMessageBox{
color=yellow;
}
QLabel{
color:yellow;
}</string>
  </property>
  <widget class="QPushButton" name="addorder">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
font:20px bold;</string>
   </property>
   <property name="text">
    <string>Add a movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeorder">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
font:20px bold;</string>
   </property>
   <property name="text">
    <string>Remove a movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showmovies">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>260</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
font:20px bold;</string>
   </property>
   <property name="text">
    <string>Show the movies</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:26pt; color:#fff000;&quot;&gt;Welcome, Admin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font:18px bold;</string>
   </property>
   <property name="text">
    <string>Director:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>349</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>350</y>
     <width>63</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font:18px bold;</string>
   </property>
   <property name="text">
    <string>Theater</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addnewmovie">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>379</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
font:20px bold;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:yellow;
font:20px bold;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:yellow;
font:18px bold;</string>
   </property>
   <property name="text">
    <string>Which theather?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="removecombobox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="modelColumn">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="removethismovie">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>551</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:20px bold;</string>
      </property>
      <property name="text">
       <string>Theater 2:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:yellow;
font:18px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>551</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:20px bold;</string>
      </property>
      <property name="text">
       <string>Theater 3:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_5">
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:yellow;
font:18px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>551</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:20px bold;</string>
      </property>
      <property name="text">
       <string>Theater 4:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_6">
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:yellow;
font:18px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>259</y>
     <width>243</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:18px bold;</string>
      </property>
      <property name="text">
       <string>Movie name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>289</y>
     <width>245</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:18px bold;</string>
      </property>
      <property name="text">
       <string>Movie genre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>319</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>551</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color:yellow;
font:20px bold;</string>
      </property>
      <property name="text">
       <string>Theater 1:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:yellow;
font:18px;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>label</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>Exit</zorder>
  <zorder>Back</zorder>
  <zorder>label_6</zorder>
  <zorder>removecombobox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>showmovies</zorder>
  <zorder>removeorder</zorder>
  <zorder>addorder</zorder>
  <zorder>removethismovie</zorder>
  <zorder>addnewmovie</zorder>
  <zorder>comboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
